natlas-server/app/templates/host/versions/0.6.10/_host-row.html

```diff
@@ -42,10 +42,6 @@ <h5 class="mt-2">Tags</h5>
         </div>
         <img class="img-responsive img-thumbnail" data-path='/media/original/{{screenshot.hash|hashpath}}' data-ip='{{host.ip}}' data-scan_id='{{host.scan_id}}' src='/media/thumbs/{{screenshot.thumb_hash|hashpath}}' alt="{{ screenshot.host }} - {{ screenshot.port }}{%if screenshot.host != host.ip %} ({{host.ip}}){%endif%}">
       </div>
-      <!--<h5 class="mt-2">Port {{ screenshot.port }}</h5>
-      <small>Host: {{screenshot.host}}</small>
-      <div class="expand-img"><img class="img-responsive img-thumbnail" data-path='/media/original/{{screenshot.hash|hashpath}}' src='/media/thumbs/{{screenshot.thumb_hash|hashpath}}' alt="{{ host.ip }} - {{ screenshot.service }} Screenshot"></div>
-      -->
       {% endif %}
     {% endfor %}
   </div><!--meta column-->
```

Comment:
This was actually pretty pervasive across all of the 0.6.10 so I've updated it. I hate this because a lot of these files don't change at all, but they _could_ change